src/firefox/index.js

```diff
@@ -134,15 +210,19 @@ export function configureProfile(
 }
 
 
+export type CreateProfileParams = {
+  app?: PreferencesAppName,
+  configureThisProfile?: ConfigureProfileFn,
+};
+
 /*
  * Creates a new temporary profile and resolves with the profile object.
  *
  * The profile will be deleted when the system process exits.
  */
 export function createProfile(
-    {app, configureThisProfile=configureProfile}: Object = {})
-    : Promise<FirefoxProfile> {
-
+  {app, configureThisProfile=configureProfile}: CreateProfileParams = {}
```

Comment:
heh, I remember now when naming `configureThisProfile` that `defaultProfileConfigurator` didn't sound right :)